src/Migrations/Version20191205223937.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20191205223937 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('DROP TABLE person_research_group_audit');
+        $this->addSql('DROP TABLE data_center_audit');
+        $this->addSql('DROP TABLE dataset_audit');
+        $this->addSql('DROP TABLE dataset_submission_audit');
+        $this->addSql('DROP TABLE dif_audit');
+        $this->addSql('DROP TABLE data_repository_audit');
+        $this->addSql('DROP TABLE distribution_point_audit');
+        $this->addSql('DROP TABLE person_data_repository_audit');
+        $this->addSql('DROP TABLE funding_cycle_audit');
+        $this->addSql('DROP TABLE person_audit');
+        $this->addSql('DROP TABLE funding_organization_audit');
+        $this->addSql('DROP TABLE person_funding_organization_audit');
+        $this->addSql('DROP TABLE research_group_audit');
+        $this->addSql('ALTER TABLE person_token ALTER valid_for TYPE VARCHAR(255)');
```

Comment:
Were these 3 alter table lines just pending?

Reply:
They are already in the previous migrations but i dont know why doctrine thinks that it needs to be run again

Reply:
This may be one of the differences between Doctrine versions.